pkg/machines/scripts/install_machine.sh

```diff
@@ -108,10 +120,9 @@ virt-install \
     $VNICS_PARAM \
     $VCPUS_PARAM \
     $BOOT_PARAM \
-    $AUTOSTART_PARAM
-EXIT_STATUS=$?
+    $AUTOSTART_PARAM || handleFailure $?
 
-if [ "$EXIT_STATUS" -eq 0 ] && vmExists "$VM_NAME"; then
+if vmExists "$VM_NAME"; then
```

Comment:
In theory this condition should now always be true, right? If you want to treat it more like an assertion, how about just calling

    vmExists "$VM_NAME"

If it fails, the script will exit due to `set -e`, and as the domain does not exist, there is nothing to clean up. Alternatively, keep an `else` branch and print a loud error message about "wah, wah, where's my VM???"